css/main.css

```
@@ -8,3 +8,43 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+footer {
+  background-color: #222;
+  width: 100%;
+  height: 65px;
+  display: flex;
+}
+
+.musicaFooter {
+  font-size: 11px;
+  color: #a5a4a4;
+  margin: 5px 0 0 5px;
+}
+
+.componenteTempo {
```

Vamos patronizar nomes de classes separados por traços, no css
```suggestion
.componente-tempo {
```